pkg/apis/extensions/v1beta1/types.go

```diff
@@ -87,6 +95,8 @@ type HorizontalPodAutoscalerSpec struct {
 	MinReplicas int `json:"minReplicas"`
 	// MaxReplicas is the upper limit for the number of pods that can be set by the autoscaler. It cannot be smaller than MinReplicas.
 	MaxReplicas int `json:"maxReplicas"`
+	// CPU utilization is the target average CPU utilization (represented as a fraction of requested CPU) over all the pods.
+	CPUUtilization *CPUTargetUtilization `json:"cpuUtilization,omitempty"`
```

Comment:
We need a corresponding field in HPAStatus, which will report the current utilization.